.husky/pre-push

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```

Comment:
For now, it's ok, but we can discuss if we want to run all checks before each push. IMO it would be enough when we run them in CI.